src/styles/isomer-cms/pages/Media.module.scss

```diff
@@ -50,6 +50,25 @@
 
       }
 
+      .mediaCardFileContainer {
```

Comment:
CSS can be simplified since it shares many similarities with `.mediaCardImageContainer`

Reply:
done